build.gradle

```diff
@@ -31,10 +31,20 @@ allprojects {
   repositories {
     mavenCentral()
     mavenLocal()
+    maven {
+      url "https://s01.oss.sonatype.org/content/repositories/snapshots/"
+      mavenContent {
+        snapshotsOnly()
```

Comment:
What is this for?

Reply:
Allows us to pull in SNAPSHOT versions. I use this during development. Makes it so I can upgrade dependencies and publish them as SNAPSHOTs and pull them in here. Leaving it in place should not cause any problems.